devtools/ci/after_sucess.sh

```diff
@@ -45,13 +45,20 @@ conda list -e
 
 (cd docs && python premake.py && make html && cd -)
 ls -lt docs/_build
+ls -lt docs/_build/html
 pwd
 echo travis_fold:end:build.docs
 
 echo travis_fold:start:upload.docs
-if [[ "$TRAVIS_BRANCH" != "master" ]]; then
+
+if [[ "$TRAVIS_BRANCH" == "master" ]]; then
+    python devtools/ci/push-docs-to-s3.py
+elif [[ "$TRAVIS_BRANCH" == "docs_deploy" ]]; then
+    # change the behavior for the docs testing branch (docs_deploy branch in
```

Comment:
interesting idea to have a branch for docs...

Reply:
yeah, that brach has actually been around for a while to test deployment behavior, since PRs from my own fork don't get the AWS keys.

Reply:
but you disabled this now, correct?

Reply:
oh, right now it behaves the same as a fork PR. But the trick is that to make that deploy (if testing why normal deployment fails), all you have to do is switch out which like is commented there. It specifically has the (commented out, but ready to use) `--clobber` option which will overwrite (instead of sync diffs). I figure that's probably what you want if you're in that situation ;-)